pages/account/manage/milestone/[[...data]].js

```diff
@@ -274,7 +277,7 @@ export default function ManageMilestone({ BASE_URL, milestone }) {
                       DELETE
                     </Button>
                   )}
-                  <Button type="submit" primary={true}>
+                  <Button className={isDisabled && "cursor-not-allowed"} type="submit" primary={true} disabled={isDisabled}>
```

Comment:
Please add this to the `Button` component, so it works everywhere `components/Button.js`

Reply:
I noticed this already exists on the component, if you use the prop `disable`